### ISSUE TITLE
const_missing with Concurrent::Array on TruffleRuby

### ISSUE THREAD

**Issue by solnic** [ruby-concurrency/concurrent-ruby#763]
Hey there 😄 I stumbled upon the following problem:

```
∞ [truffleruby-1.0.0-rc6][node-v7.3.0] core git(master) ruby -v
truffleruby 1.0.0-rc6, like ruby 2.4.4, GraalVM CE Native [x86_64-darwin]

∞ [truffleruby-1.0.0-rc6][node-v7.3.0] core git(master) irb
irb(main):001:0> require 'concurrent/array'
=> true
irb(main):002:0> Concurrent::Array
NameError: uninitialized constant Concurrent::Array
Did you mean?  Array
        from (irb):2:in `const_missing'
        from (irb):2
        from /Users/solnic/.rubies/truffleruby-1.0.0-rc6/bin/irb:29:in `<main>'
```

`concurrent` version is `1.0.5`

**Comment by pitr-ch** [comment]
Thanks for letting us know! This is already fixed in `concurrent-ruby` 1.1.0.pre1 (however there is similar problem with Set until pre2 is out.)

**Comment by solnic** [comment]
@pitr-ch thanks. Are you not planning to backport the fix to 1.0.x?

**Comment by pitr-ch** [comment]
No at the moment, the 1.1 is fully compatible.
